src/middlewared/middlewared/plugins/device_/device_info_base.py

```diff
@@ -0,0 +1,39 @@
+import re
+
+from middlewared.service import private, ServicePartBase
+
+RE_SERIAL_NUMBER = re.compile(r'Serial Number:\s+(?P<serial>.+)', re.I)
+
+
+class DeviceInfoBase(ServicePartBase):
+
+    RE_SERIAL_NUMBER = RE_SERIAL_NUMBER
```

Comment:
I dont quite understand why declare again in the class instead of just define it here to begin with?